CHANGES.md

```diff
@@ -1,3 +1,8 @@
+# 3.0.0 UNRELEASED
+- [FIXED] Expose `BasePlugin` type in Cloudant client.
+- [REMOVED] Remove nodejs-cloudant TypeScript type definitions for `db.search`.
```

Comment:
Does this change needs to explain that the Nano search types are used instead?

Reply:
Yeah, I'll add some detail.